apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -272,37 +278,57 @@ var TopInstructions = React.createClass({
             }
           </div>
         </PaneHeader>
-        {!this.state.helpTabSelected &&
-          <div style={[this.props.collapsed && commonStyles.hidden]}>
-            <div style={styles.body}>
-              {this.props.hasContainedLevels && <ContainedLevel ref="instructions"/>}
-              {!this.props.hasContainedLevels &&
-                <div ref="instructions">
+        <div style={[this.props.collapsed && commonStyles.hidden]}>
+          <div style={styles.body}>
+            {this.props.hasContainedLevels && <ContainedLevel ref="instructions"/>}
+            {!this.props.hasContainedLevels &&
+              <div ref="instructions">
+                {!this.state.helpTabSelected &&
                   <Instructions
                     ref="instructions"
                     renderedMarkdown={processMarkdown(this.props.markdown,
                       { renderer })}
                     onResize={this.adjustMaxNeededHeight}
                     inTopPane
                   />
-                  <TeacherOnlyMarkdown/>
-                </div>
-              }
-            </div>
-            {!this.props.isEmbedView &&
-              <HeightResizer
-                position={this.props.height}
-                onResize={this.handleHeightResize}
-              />}
-          </div>
-        }
-        {this.state.helpTabSelected &&
-          <div style={[this.props.collapsed && commonStyles.hidden]}>
-            <div style={styles.body}>
-              Help Tab
-            </div>
+                }
+                {this.state.helpTabSelected &&
+                  <div id="helpTab">
+                    <div className="reference_area" id="reference_area">
```

Comment:
do we need this id/className for anything?

Reply:
It looks like there is some NetSim code that uses the id. Will remove the className.

Reply:
The reason I like to avoid ids/classNames in React components is it immediately makes your component less isolated, as code anywhere else in your app could be doing a getElementById and messing with things. An isolated component is one that is much easier to understand and reason about :)